test/util.spec.js

```diff
@@ -45,21 +45,39 @@ describe('util', () => {
   })
 
   it('.serialize and .deserialize large objects', (done) => {
-    const size = 65536 * 2
+    // larger than the default borc heap size, should auto-grow the heap
+    const size = 1 << 17 // 128Kb
     const largeObj = { someKey: [].slice.call(new Uint8Array(size)) }
-    // Configure decoder with custom size
-    dagCBOR.util.configureDecoder({ size: size + 1 })
 
     dagCBOR.util.serialize(largeObj, (err, serialized) => {
       expect(err).to.not.exist()
-      expect(Buffer.isBuffer(serialized)).to.equal(true)
+      expect(Buffer.isBuffer(serialized)).to.be.true()
 
       dagCBOR.util.deserialize(serialized, (err, deserialized) => {
         expect(err).to.not.exist()
         expect(largeObj).to.eql(deserialized)
+        // reset decoder to default
+        dagCBOR.util.configureDecoder({ size: 1 << 16, maxSize: 1 << 26 })
```

Comment:
Could the code be changed so that a call without any options would reset the decoder to the default values? I think that would make it nicer to use and less error-prone in case we change the default values at one point.